config/application.rb

```diff
@@ -17,6 +17,7 @@ class Application < Rails::Application
     config.i18n.fallbacks = true
 
     config.middleware.use "Redirector" unless Rails.env.development?
+    config.middleware.use Rack::Attack
```

Comment:
```
Run options: --seed 22073

# Running:

................................................................
throttled
E......................................................................................................................
throttled
E
throttled
E
throttled
E
throttled
E
throttled
E.............................................S.................................................................................................................................
................................................................................................................................................................................
.......................................................................................................................S........................................................
.................................................S..............................................................................................................................
.............................................................................................

Finished in 87.942829s, 11.1891 runs/s, 20.9795 assertions/s.

  1) Error:
PasswordResetTest#test_resetting_a_password_when_signed_in:
Capybara::ElementNotFound: Unable to find field "Password"
    test/integration/password_reset_test.rb:76:in `block in <class:PasswordResetTest>'


  2) Error:
ProfileTest#test_changing_to_an_existing_handle:
Capybara::ElementNotFound: Unable to find link "Edit Profile"
    test/integration/profile_test.rb:33:in `block in <class:ProfileTest>'


  3) Error:
ProfileTest#test_changing_email_allows_signing_in_with_new_email:
Capybara::ElementNotFound: Unable to find link "Edit Profile"
    test/integration/profile_test.rb:44:in `block in <class:ProfileTest>'


  5) Error:
ProfileTest#test_disabling_email_on_profile:
Capybara::ElementNotFound: Unable to find link "Edit Profile"
    test/integration/profile_test.rb:58:in `block in <class:ProfileTest>'


  6) Error:
ProfileTest#test_adding_Twitter_username:
Capybara::ElementNotFound: Unable to find link "Edit Profile"
    test/integration/profile_test.rb:71:in `block in <class:ProfileTest>'

984 runs, 1845 assertions, 0 failures, 6 errors, 3 skips

You have skipped tests. Run with --verbose for details.
```

Log from sample test run with middleware for test env with changed initializer to use `payload` and print when it is throttled.

```ruby
  ActiveSupport::Notifications.subscribe('rack.attack') do |_name, _start, _finish, _request_id, payload|
    if payload.env['rack.attack.match_type'] == :throttle
      data = {
        status: 'throttled',
        ip: payload.ip.to_s,
        method: payload.env["REQUEST_METHOD"],
        path: payload.env["REQUEST_PATH"],
        matched: payload.env["rack.attack.matched"],
        discriminator: payload.env["rack.attack.match_discriminator"],
        match_data: payload.env["rack.attack.match_data"]
      }
      Rails.logger.info data
      print "\nthrottled\n"
    end
  end
```